packages/taquito-signer/src/import-key.ts

```diff
@@ -12,15 +11,15 @@ import { TezosToolkit } from '@taquito/taquito';
  * @param secret Faucet secret
  */
 export async function importKey(
-  toolkit: TezosToolkit,
+  toolkit: any,
```

Comment:
Not ideal to set this to any

Reply:
I agree with you, however, it seems like would just be necessary while we wait for removing the signer package from the main taquito package. 

Maybe an acceptable solution is to put a TODO over this and add a new task in this issue https://github.com/ecadlabs/taquito/issues/315, That way we can remember to remove it and put back the type.

What do you think?

Reply:
I think this is acceptable for the short term.

We can also consider removing the method (#315) in 6.2 instead of waiting for v7.